examples/case_studies/bart_heteroscedasticity.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        hdi_data=w_hdi["w"].sel(w_dim_0=0),
</pre><p>This is more explicit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/519/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>fixed in https://github.com/pymc-devs/pymc-examples/pull/519/commits/a5b2acdf0ef1462da61531e575569adc045db660 :D</p>